plugins/modules/k8s.py

```diff
@@ -87,6 +87,7 @@
       will only work if the same object is passed with state=absent (alternatively, just use state=absent with the name including
       the generated hash and append_hash=no)
     - Requires openshift >= 0.7.2
+    default: False
```

Comment:
@goneri Just to verify: the modules are applying a default of `False` but it wasn't documented?

Reply:
This line was actully coming from: https://github.com/ansible-collections/community.kubernetes/pull/325